build/raptor-test-transformer.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+/**
+ * Overview:
+ *
+ * 1. Find all '.js' files in the app directory
+ * 2. Replace the path with the '.esp.js' one
+ * 3. Check if such file exists
+ * 4. Run this transformer + espect to apply the esp file on to target file
+ */
+var utils = require('utils');
+
+exports.execute = function(options) {
+  var webapp = options.webapp;
+  var debuguypath = options.RAPTOR_TRANSFORMER_PATH;
+  if (!debuguypath) {
+    debuguypath = options.GAIA_DIR + '/node_modules/debuguy';
```

Comment:
this script will be executed in xpcshell environment so I recommend that handle path by utils.joinPath() for better compatible with windows.